### ISSUE TITLE
Bug: `anaconda notebook download` fails with `BinstarError`

### ISSUE THREAD

**Issue by stephenakearns** [Anaconda-Server/anaconda-client#250]
logged by @electronwill : 

Downloading notebooks with the web interface works but 'anaconda notebook download brief-test' and 'anaconda notebook download wwarner/brief-test' both fail with "BinstarError"

**Comment by stephenakearns** [comment]
whoever ends up working on this, please check with @malev as he will likely have the most context on the Anaconda notebook commands in the Client tool

**Comment by stephenakearns** [comment]
@electronwill - can you copy and paste in the actual screen output with the error message? (as much detail as possible)

**Comment by electronwill** [comment]
Sure. This was from https://github.com/Anaconda-Server/anaconda-server/issues/823

'anaconda notebook download brief-test' and 'anaconda notebook download wwarner/brief-test' both fail with BinstarError:

```
$ anaconda notebook download brief-test
Using anaconda-server api site http://52.91.112.115:8080/api
[BinstarError] ('OK: Request fulfilled, document follows ([GET] http://52.91.112.115:8080/api/download/wwarner/brief-test/2015.11.23.1730/brief-test.ipynb -> 200)', 200)
$
```

No file is created.

**Comment by electronwill** [comment]
I just realized that this error happened when using Alan's test server for the Ford project at http://54.165.195.42:8080 , but when using Anaconda Cloud it works fine.

**Comment by dsludwig** [comment]
This has been fixed in the latest release (1.2.1)
